### ISSUE TITLE
parámetro ARENA_INICIAL

### ISSUE THREAD

**Issue by TriniAguilera** [IIC2233/Syllabus#132]
### Prerrequisitos
(Marcar colocando una X entre los corchetes los ítems que ya hiciste, así: "[X]")

* [X ] Leí las reglas del foro (https://github.com/IIC2233/Syllabus/wiki/8.-Reglas-del-foro-(issues)).
* [X ] Busqué en las *issues* si ya preguntaron mi duda y no encontré nada parecido (https://github.com/IIC2233/syllabus/issues/).
* [X ] Revisé el compilado de dudas de la tarea y no encontré una *issue* similar a la mía ([click aquí para ir al compilado](/IIC2233/syllabus/issues/127)).
* [X ] Mi duda no se trata sobre una librería, *built-in* o mala práctica, ya que eso se pregunta en la *issue* creada para ello ([click aquí para ir a esa _issue_](/IIC2233/syllabus/issues/126)).
* [X ] Mi duda no se trata de un tema administrativo o personal, ya que en ese caso debo contactar a mi profe, al ayudante de Bienestar o al correo del curso (iic2233@ing.puc.cl).
* [X ] Utilizaré un título descriptivo y llenaré correctamente esta plantilla.
* [X ] De ser necesario, colocaré código simple que permita explicar mi problema o duda, ya que compartir código de esta evaluación puede ser considerado una falta a la ética.

Entiendo que el parámetro ARENA_INICIAL entrega el tipo de arena con el que se tiene que comenzar el torneo. Mi duda es que si se comienza siempre con una arena de tipo Normal o puede ser cualquiera de los existentes. A partir de eso, podría elegir la arena entre las que hay disponibles y que esta sea del mismo tipo del que indica el parámetro

**Comment by osillano** [comment]
Hola! Puede ser cualquier tipo de arena. El parámetro `ARENA_INICIAL` debe ser un `str` entre los siguientes: 'normal', 'mojada', 'rocosa' o 'magnetica'. Al crear una nueva partida se debe elegir aleatoriamente una arena que sea del tipo especificado por `ARENA_INICIAL`.

**Comment by tremendapithon** [comment]
Hola! Tengo una duda respecto en la parte en donde tenemos que escoger de forma aleatoria en arena_inicial, ya que pienso que estos elementos deben venir del archivo arena.csv, tipo al comenzar una nueva partida escojo algún tipo de arena del archivo "arenas.csv" y de ahi considero todos sus atributos??

**Comment by osillano** [comment]
Sí, al comenzar una partida se debe escoger aleatoriamente una de las arenas listadas en `arenas.csv` tal que el tipo de esta arena coincida con `ARENA_INICIAL`.
